packages/contract-wrappers/src/generated-wrappers/broker.ts

```diff
@@ -522,7 +522,7 @@ export class BrokerContract extends BaseContract {
 
     public getABIDecodedTransactionData<T>(methodName: string, callData: string): T {
         const functionSignature = this.getFunctionSignature(methodName);
-        const self = (this as any) as BrokerContract;
```

Comment:
we shouldn't format generated files. 

could we add `generated-wrappers` in `.prettierignore`?